src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`2.IVsContainedLanguageFactory.cs

```diff
@@ -67,6 +68,11 @@ private AbstractProject FindMatchingProject(IVsHierarchy hierarchy, uint itemid)
         }
 
         public int GetLanguage(IVsHierarchy hierarchy, uint itemid, IVsTextBufferCoordinator bufferCoordinator, out IVsContainedLanguage language)
+        {
```

Comment:
this is old API (contained language xxx API), so probably not needed, but in case we want to provide documentservicefactory here as well for old API code path.